tests/cli/test_compose_qcow2.sh

```diff
@@ -13,12 +13,10 @@ set -e
 . $(dirname $0)/lib/lib.sh
 
 CLI="${CLI:-./src/bin/composer-cli}"
-QEMU_BIN="/usr/bin/qemu-system-$(uname -m)"
-QEMU="$QEMU_BIN -machine accel=kvm:tcg"
 
 rlJournalStart
     rlPhaseStartSetup
-        rlAssertExists $QEMU_BIN
+        rlAssertExists $(which $QEMU)
```

Comment:
If you copy the deleted QEMU and QEMU_BIN variables and adjust their values in the cherry-picked commit all of these assert changes will not be needed.